packages/core/src/messagingProtocols/proteus/ProteusService/CryptoClient/CoreCryptoWrapper/CoreCryptoWrapper.ts

```diff
@@ -177,8 +177,35 @@ export class CoreCryptoWrapper implements CryptoClient {
     return this.coreCrypto.proteusCryptoboxMigrate(dbName);
   }
 
+  /**
+   * Will call the callback once corecrypto is ready.
+   * @param callback - Function to be called once corecrypto is ready.
+   * @see https://github.com/wireapp/wire-web-packages/pull/4972
+   */
+  private async onReady(callback: () => Promise<void>) {
+    const isCoreCryptoReady = async () => {
+      const strongRefCount = this.coreCrypto.strongRefCount();
+      const isCoreCryptoReady = strongRefCount <= 1;
+      return isCoreCryptoReady;
+    };
```

Comment:
```suggestion
    const isCoreCryptoReady = () => {
      const strongRefCount = this.coreCrypto.strongRefCount();
      const isCoreCryptoReady = strongRefCount <= 1;
      return isCoreCryptoReady;
    };
```
There is no await in there it seems